com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -89,6 +89,16 @@ public class Configurations {
     public partial class UIWidgetsPanel : RawImage, IUIWidgetsWindow {
         static List<UIWidgetsPanel> panels = new List<UIWidgetsPanel>();
 
+        static public Isolate anyIsolate {
+            get {
+                if (panels.Count == 0) {
+                    return null;
+                }
+
+                return panels[0]._wrapper.isolate;
```

Comment:
what if there are more than one active panels?  what will it be if the panel that calls the OnUIWidetsMethodMessage is panel[1] ?

Reply:
This is used to make sure the method is excuted. "Updating view matrix" does not need subspecific panel. For updating text, the corresponding isolate will be gotten scope in its function.